src/config-schema.js

```diff
@@ -262,6 +262,11 @@ const configSchema = {
         type: 'boolean',
         default: true
       },
+      restorePreviousWindowsOnStartAlways: {
+        description: 'When checked *ALWAYS* restores the last state of all Atom windows.',
```

Comment:
@BrunoBernardino, actually, it could be more than one file, like: `atom fileX fileY fileZ` and it will restore the last session + there will open the 3 files, so if the short "just always" version is not quite clear, maybe add at the end ", no matter how atom is started".